.github/workflows/data_library_quarterly.yml

```diff
@@ -61,6 +65,17 @@ jobs:
           AWS_ACCESS_KEY_ID: "op://Data Engineering/DO_keys/AWS_ACCESS_KEY_ID"
 
       - uses: NYCPlanning/action-library-archive@v1.1
+        if: ${{ !(inputs.use_staging_image) }}
+        with:
+          name: ${{ matrix.dataset }}
+          version: ${{ github.event.inputs.version }}
+          latest: true
+          s3: true
+          compress: true
+          output_format: pgdump shapefile csv
+
+      - uses: NYCPlanning/action-library-archive@v1.2
+        if: ${{ inputs.use_staging_image }}
```

Comment:
Did not add the issue creation to this one since it is run manually when the quarterly updates are put out